src/ds5/ds5-auto-calibration.cpp

```diff
@@ -418,7 +426,7 @@ namespace librealsense
             _json = json;
             _action = auto_calib_action::RS2_OCC_ACTION_ON_CHIP_CALIB;
             _interactive_state = interactive_calibration_state::RS2_OCC_STATE_WAIT_TO_CAMERA_START;
-            _interactive_scan = bool(interactive_scan_v3);
+            _interactive_scan = false;//bool(interactive_scan_v3); - enforce non-interactive run
```

Comment:
Remove the old part of the comment

Reply:
And explain why non iterative run is enforced shortly in comment

Reply:
done